extensions/resource-deployment/src/ui/deployAzureSQLDBWizard/pages/databaseSettingsPage.ts

```diff
@@ -103,10 +103,22 @@ export class DatabaseSettingsPage extends BasePage {
 				value: constants.IpAddressInfoLabel
 			}).component();
 
+		// regex for validation
+		let ipRegex = /(^(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$)/;
+
 		//Start IP Address Section:
 
 		this._startIpAddressTextbox = view.modelBuilder.inputBox().withProperties(<azdata.InputBoxProperties>{
-			inputType: 'text'
+			inputType: 'text',
+			required: true,
+			validationErrorMessage: localize('deployAzureSQLDB.DBMaxIpInvalidError', "Max Ip address is invalid")
```

Comment:
Shouldn't this be "min" instead of "max"? I don't think you need to specify that this is the "min" address anyways - the error message shows up attached to the box so it should be clear which one it applies to. `Invalid IP address format` should be fine.

Reply:
Max/min for Ip address is a weird notion.